### ISSUE TITLE
Error using chunk

### ISSUE THREAD

**Issue by fubuloubu** [fission-suite/ts-sdk#63]
The following occurs when I try to run:
```
var wnfs = await FileSystem.empty();
const cid = await wnfs.mkdir("private/gunclear/safe")
```
Error:
```
Unhandled Rejection (TypeError): o[Symbol.iterator] is not a function
./node_modules/fission-sdk/ipfs/basic.js/</__asyncValues<
http://localhost:3000/static/js/1.chunk.js:105016:96
./node_modules/fission-sdk/ipfs/basic.js/</add/</<
src/ipfs/basic.ts:11

   8 |     });
   9 | };
  10 | var __generator = (this && this.__generator) || function (thisArg, body) {
> 11 |     var _ = { label: 0, sent: function() { if (t[0] & 1) throw t[1]; return t[1]; }, trys: [], ops: [] }, f, y, t, g;
  12 |     return g = { next: verb(0), "throw": verb(1), "return": verb(2) }, typeof Symbol === "function" && (g[Symbol.iterator] = function() { return this; }), g;
  13 |     function verb(n) { return function (v) { return step([n, v]); }; }
  14 |     function step(op) {

step
http://localhost:3000/static/js/1.chunk.js:104996:17
./node_modules/fission-sdk/ipfs/basic.js/</__generator</verb/<
http://localhost:3000/static/js/1.chunk.js:104927:14
fulfilled
```

**Comment by icidasset** [comment]
Thanks for creating this issue 🙏 
I think this is an issue with the latest js-ipfs `v0.48`
Are you loading ipfs by yourself, or letting the sdk handle that? (the sdk automatically loads the latest version)

**Comment by fubuloubu** [comment]
SDK yeah

It might be a good idea to have this log message tell you what version it loaded:
```
window.ipfs is not available, loading js-ipfs...          get-ipfs.ts:67
```

**Comment by icidasset** [comment]
Yeah good idea 👍  In the meanwhile, you can see in the network tab which version it loads.
Should load it from `unpkg.com`, if you don't have ipfs companion.

**Comment by fubuloubu** [comment]
Yup, 0.48.0

**Comment by icidasset** [comment]
We'll create a new release with a fix soon.
In the meantime you can tell the SDK to load the previous IPFS version by doing:
```js
fissionSdk.ipfs.getIpfs({ jsIpfs: "https://unpkg.com/ipfs@0.47.0/dist/index.min.js" })
```
Run that before anything else you do with the sdk.

Resources:
- [getIpfs options](https://github.com/fission-suite/get-ipfs#config)

**Comment by icidasset** [comment]
Released `v0.11` with a fix for js-ipfs v0.48

Additionally:
- Added more info to the readme about how to this sdk
- The loaded js-ipfs version is now set to `0.48` specifically
